dmoj/executors/java_executor.py

```diff
@@ -13,13 +13,11 @@
 from dmoj.utils.unicode import utf8bytes, utf8text
 from .base_executor import CompiledExecutor
 
-recomment = re.compile(br'/\*.*?\*/', re.DOTALL)
-restring = re.compile(br''''(?:\\.|[^'\\])'|"(?:\\.|[^"\\])*"''', re.DOTALL)
-reinline_comment = re.compile(br'//.*?(?=[\r\n])')
-reclass = re.compile(br'\bpublic\s+(?:strictfp\s+)?(?:(?:abstract|final)\s+)?(?:strictfp\s+)?class\s+([_a-zA-Z\$][_0-9a-zA-z\$]*?)\b')
-repackage = re.compile(br'\bpackage\s+([^.;]+(?:\.[^.;]+)*?);')
-redeunicode = re.compile(br'\\u([0-9a-f]{4})', re.I)
-deunicode = lambda x: redeunicode.sub(lambda a: six.unichr(int(a.group(1), 16)), x)
+recomment = re.compile(r'/\*.*?\*/', re.DOTALL)
+restring = re.compile(r''''(?:\\.|[^'\\])'|"(?:\\.|[^"\\])*"''', re.DOTALL)
+reinline_comment = re.compile(r'//.*?(?=[\r\n])')
+reclass = re.compile(r'\bpublic\s+(?:strictfp\s+)?(?:(?:abstract|final)\s+)?(?:strictfp\s+)?class\s+([\w\$][\w\$]*?)\b',re.U)
```

Comment:
Need a space before `re.U`. You should probably make all of them `re.U` for consistently, since on python 3, that's the default, and we don't want python 2 to run differently.